src/views/home/Home.vue

```diff
@@ -1,88 +1,87 @@
+          v-show="filter === '' || edition.name.toLowerCase().indexOf(filter.toLowerCase()) !== -1"
+          :key="edition.versionId">
+        <edition-card :edition="edition"></edition-card>
+      </li>
+    </ul>
+    <div class="row">
+      <div class="col">
+        <small>{{ $tc('home.publicEditionGroupCount', numberOfEditions)}}</small>
       </div>
-      <ul class="list-unstyled row mt-2" id="search-results" v-if="allScrolls.length">
-        <li 
-            class="col-sm-6 col-md-4 col-lg-3 col-xl-2 mb-3 list-item" 
-            v-for="scroll in allScrolls" 
-            v-show="filter === '' || scroll.name.toLowerCase().indexOf(filter.toLowerCase()) !== -1" 
-            :key="scroll.id">
-          <scroll-card :scroll="scroll"></scroll-card>
-        </li>
-      </ul>
     </div>
+    <ul class="list-unstyled row mt-2" id="all-search-results" v-if="allEditions.length">
+      <li
+          class="col-sm-6 col-md-4 col-lg-3 col-xl-2 mb-3 list-item"
+          v-for="edition in allEditions"
+          v-show="filter === '' || edition.name.toLowerCase().indexOf(filter.toLowerCase()) !== -1"
+          :key="edition.id">
+        <edition-group-card :edition="edition"></edition-group-card>
+      </li>
+    </ul>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import Waiting from '@/components/misc/Waiting.vue';
-import ScrollService from '@/services/scroll';
-import ScrollCard from './components/ScrollCard.vue';
-import ScrollVersionCard from './components/ScrollVersionCard.vue';
-import { ScrollInfo, ScrollVersionInfo } from '@/models/scroll';
+import EditionService from '@/services/edition';
+import EditionGroupCard from './components/EditionGroupCard.vue';
+import EditionCard from './components/EditionCard.vue';
+import { EditionInfo } from '@/models/edition';
 import { countIf } from '@/utils/helpers';
 
 export default Vue.extend({
   name: 'home',
   components: {
     Waiting,
-    ScrollCard,
-    ScrollVersionCard,
+    EditionGroupCard,
+    EditionCard,
   },
   data() {
     return {
-      scrollService: new ScrollService(this.$store),
-      allScrolls: [] as ScrollInfo[],
-      myScrolls: [] as ScrollVersionInfo[],
+      editionService: new EditionService(this.$store),
+      allEditions: [] as EditionInfo[],
+      myEditions: [] as EditionInfo[],
       filter: '',
     };
   },
   computed: {
-    numberOfScrolls(): number {
-      return countIf(this.allScrolls, (scroll) => this.nameMatch(scroll.name));
+    numberOfEditions(): number {
+      return countIf(this.allEditions, (edition) => this.nameMatch(edition.name));
     },
-    numberOfMyScrolls(): number {
-      return countIf(this.myScrolls, (scrollVer) => this.nameMatch(scrollVer.name));
+    numberOfMyEditions(): number {
+      return countIf(this.myEditions, (edition) => this.nameMatch(edition.name));
     }
   },
   mounted() {
     // We do not use async/await here because we want both requests to go out simultaneously.
```

Comment:
The bigger fear with async/await is that they perhaps shouldn't be used in Vue lifecycle hooks.  If you do want simultaneous async requests `await Promise.all(asyncFunc1(), asyncFunc2(), ...)` is pretty cool, if you haven't tried it already.  I see though that the comment is now irrelevant.

Reply:
I think I didn't use `await Promise.all` because it loses the return value of the calls, I'm not sure.

Reply:
Yeah, if you want to do it with return values, you can always use destructuring :)
```JS
const async1 = async(thing) => {return thing}
const async2 = async(thing) => {return thing}

const doBoth = async() => {
  let [thing1, thing2] = await Promise.all([async1("thing1"), async2("thing2")])
  console.log(thing1)
  console.log(thing2)
}

doBoth()
```